pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -228,7 +226,6 @@ func validateSource(ctx context.Context, client kubecli.KubevirtClient, vmClone
 	if source.Kind != "" && source.Name != "" {
 		switch source.Kind {
 		case virtualMachineKind:
-			causes = append(causes, validateCloneSourceVM(ctx, client, source.Name, vmClone.Namespace, sourceField.Child("Source"))...)
 		case virtualMachineSnapshotKind:
 			causes = append(causes, validateCloneSourceSnapshot(ctx, client, source.Name, vmClone.Namespace, sourceField.Child("Source"))...)
```

Comment:
I think that we should be consistent here. If we wait for VMs to be created, we probably also wait for snapshots to do so. However I guess we can do that in a follow-up.

Reply:
You're one step ahead of me :smile: 
Seems like this is already addressed here: https://github.com/kubevirt/kubevirt/pull/13831.